numpy/add_newdocs.py

```diff
@@ -463,6 +463,65 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Create a tuple of `nditer`s which each iterate over different axes of an
+    array. They are ordered. Advancing one will change the subsequent
```

Comment:
Instead of "they are ordered", maybe "They should be iterated in nested loops, and are ordered with the iterator to be used by the outermost loop first"?

Reply:
"Create a tuple of `nditer`s which iterate in nested loops over different axes of the `op` argument. Advancing one will change the subsequent ..." ?

Reply:
I think we should make it clear that the intended use is that first iterator goes in the outer loop, and the last in the inner. Right now, you sort of have to deduce that from the behavior description of "will change subsequent".

Reply:
if there are more than two?

Reply:
"Create a tuple of nditers which iterate in nested loops over different axes of the op argument. The first iterator is used in the outermost loop, the last in the innermost loop. Advancing one will change the subsequent ..."

Reply:
`s/(outer|inner)/$1most/g`